spec/models/miq_queue_spec.rb

```diff
@@ -4,6 +4,7 @@
   context "#deliver" do
     before do
       _, @miq_server, @zone = EvmSpecHelper.create_guid_miq_server_zone
+      MiqServer.define_singleton_method(:setup_data_directory) {}
```

Comment:
@djberg96 looks a little weird to define a new method on MiqServer just so that a test can stub it and return an exception.

Could you have the test queue a real method on MiqServer but stub it and raise the exception?

Reply:
@agrare Agreed, updated.